geomesa-arrow/geomesa-arrow-dist/pom.xml

```diff
@@ -23,107 +29,40 @@
             <groupId>org.locationtech.geomesa</groupId>
             <artifactId>geomesa-arrow-tools_${scala.binary.version}</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.locationtech.geomesa</groupId>
-            <artifactId>geomesa-tools_${scala.binary.version}</artifactId>
-            <classifier>data</classifier>
-        </dependency>
 
-        <!-- GeoServer Plugins -->
+        <!-- GeoServer plugin - needed as a dependency for reactor ordering, but don't bring in transitive deps -->
```

Comment:
Hmm... I wonder if there's something else we could be depending on.  This is slick which is both cool and worrying;).

Reply:
the problem is due to us not attaching the gs-plugin tars to maven anymore... so we have to include the gs-plugin as a <file> instead of a <dependency> in the maven assembly plugin. but we still need the reactor order to be correct... so yeah, i'm not sure of a better way without re-attaching the gs-plugin tars (which would mean the get deployed to central again)